store/local_cache_supplier_reactions.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package store
+
+import (
+	"context"
+
+	"github.com/mattermost/platform/model"
+)
+
+func (s *LocalCacheSupplier) handleClusterInvalidatePost(msg *model.ClusterMessage) {
+	s.reactionCache.Remove(msg.Data)
+}
+
+func (s *LocalCacheSupplier) ReactionSave(ctx context.Context, reaction *model.Reaction, hints ...LayeredStoreHint) *LayeredStoreSupplierResult {
+	doInvalidateCacheCluster(s.reactionCache, reaction.PostId)
```

Comment:
Yes I agree it's a bit too many cluster messages for this case. We should eventually handle this where we collate these low priority invalidation messages. However this is how it worked before so it's fine for this PR.